cypress/cypress/e2e/submariner_uninstall.cy.js

```diff
@@ -0,0 +1,46 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
+ * Copyright (c) 2022 Red Hat, Inc.
+ ****************************************************************************** */
+
+/// <reference types="cypress" />
+
+import { clusterSetMethods } from '../views/clusterset/clusterset'
+
+describe('submariner - uninstall validation', {
+    tags: ['@submariner', '@e2e'],
+    retries: {
+        runMode: 0,
+        openMode: 0,
+    }
+}, function () {
+    before(function () {
+        cy.setAPIToken()
+        cy.clearOCMCookies()
+        cy.login()
+        
+    })
+
+    it('test the Uninstall Submariner add-ons', { tags: ['uninstall'] }, function () {
+        let clusterSetName = Cypress.env('CLUSTERSET_NAME')
+        // verify that a cluster set with deployment of submariner is exists.
+        clusterSetMethods.clusterSetShouldExist(clusterSetName)
+        cy.get('[data-label=Name]').contains(clusterSetName).then($value => {
+            length = $value.length
+
+            if (length != 1) this.skip()
+            else {
+                cy.get('[data-label=Name]').eq(1).click(15, 30)
+                cy.get('.pf-c-nav__link').contains('Submariner add-ons').click()
+
+                cy.get('.pf-c-table__check > label > input').click({multiple: true}).should('be.checked')
+                cy.get('#toggle-id').click()
+                cy.get('.pf-c-dropdown__menu-item').should('be.visible').click()
+                cy.get('.pf-c-form__actions > .pf-m-primary').click()
+
+                cy.get('[data-label="Cluster"]', {timeout: 180000}).should('not.exist')
```

Comment:
@NogaTal98 
Overall it looks good.
But when I tried the uninstall test on my environment, it failed.

It failed because the test reached the timeout and one of the clusters was still in the process of uninstalling.

```
<failure message="Timed out retrying after 180000ms: Expected [ &lt;th.pf-m-nowrap.pf-c-table__sort.pf-m-selected&gt;, 1 more... ] not to exist in the DOM, but it was continuously found." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 180000ms: Expected [ <th.pf-m-nowrap.pf-c-table__sort.pf-m-selected>, 1 more... ] not to exist in the DOM, but it was continuously found.
```


I wonder if we should increase the timeout more or maybe there could be another way to check for the uninstall process.